x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -476,7 +476,7 @@ filebeat.inputs:
 [float]
 ==== `request.keep_alive.disable`
 
-This specifies whether to disable keep-alives for HTTP end-points. Default: `true`.
+This specifies whether to disable keep-alive for HTTP end-points. Default: `true`.
```

Comment:
I don't think this change is correct.

Reply:
I thought I should keep it "keep-alive" to keep it as it is what is passed to the header for reuse of a connection. However, after you mentioned it, I checked https://github.com/golang/go/blob/master/src/net/dial.go and it appears that "keep-alives" is something that is also used there. Reverting this change.

Reply:
Yes, "keep-alive" is a countable noun.